tests/integration/integration.js

```diff
@@ -199,6 +199,143 @@ tap.test(
             })
         }
       )
+
+      t.test(
+        'should lookup data using query param, validate and \
+        transform, then map and send response to external service and finally outputting XML response using a get request',
+        async t => {
+          t.plan(5)
+
+          const fhirPatient = {
+            resourceType: 'Patient',
+            gender: 'unknown'
+          }
+
+          server.on('request', async (req, res) => {
+            if (req.method === 'GET' && req.url === '/fhir/Patient') {
+              t.pass()
+              res.writeHead(201, {'Content-Type': 'application/json'})
+              res.end(JSON.stringify(fhirPatient))
+              return
+            }
+            if (req.method == 'POST' && req.url === '/api') {
+              req.on('data', chunk => {
+                t.equals(
+                  chunk.toString(),
+                  '{"sex":"U","nationality":"South Africa"}'
+                )
+              })
+
+              res.writeHead(201, {'Content-Type': 'application/json'})
+              res.end()
+              return
+            }
+            res.writeHead(404)
+            res.end()
+            return
+          })
+
+          const testEndpoint = {
+            name: 'Integration Test Endpoint 2',
+            endpoint: {
+              pattern: '/integrationTest2'
+            },
+            transformation: {
+              input: 'JSON',
+              output: 'XML'
+            },
+            constants: {
+              nationality: 'South Africa'
+            },
+            requests: {
+              lookup: [
+                {
+                  id: 'fhir-server',
+                  config: {
+                    method: 'get',
+                    url: `http://localhost:${mockServerPort}/fhir/Patient`,
+                    headers: {
+                      'Content-Type': 'application/json'
+                    }
+                  },
+                  allowedStatuses: ['2xx']
+                }
+              ],
+              response: {
+                id: 'MPI',
+                config: {
+                  method: 'post',
+                  url: `http://localhost:${mockServerPort}/api`,
+                  headers: {
+                    'Content-Type': 'application/json'
+                  }
+                }
+              }
+            },
+            inputValidation: {
+              type: 'object',
+              properties: {
+                lookupRequests: {
+                  type: 'object',
+                  properties: {
+                    'fhir-server': {
+                      type: 'object',
+                      properties: {
+                        gender: {
+                          type: 'string',
+                          enum: ['female', 'unknown']
```

Comment:
we are missing some options here as described within the inputMapping transforms function options
```suggestion
                          enum: ['male', 'female', 'other', 'unknown']
```